src/index.jsx

```diff
@@ -46,6 +46,7 @@ initialize({
       mergeConfig({
         SUPPORT_URL: process.env.SUPPORT_URL || null,
         SUPPORT_EMAIL: process.env.SUPPORT_EMAIL || null,
+        LEARNING_BASE_URL: process.env.LEARNING_BASE_URL || 'http://localhost:2000',
```

Comment:
as `.env` is the default for all - why do you think you need to add `'http://localhost:2000',` here?
https://create-react-app.dev/docs/adding-custom-environment-variables/#what-other-env-files-can-be-used

Reply:
+1 I'm also confused by it.
My guess is, that it's the same as for the other settings - to provide a meaningful value when no setting is set.
I can't think of many scenarios where it would be harmful, other than refactoring, because it would require to change the url in multiple places, but that's easy to find by grepping.
And at the same time, the scenarios where it would be helpful, that I can think of, are when you set the value for that setting in the `.env` of higher priority (e.g. `.env.development.local`) to an empty string, which is rear, but can be a pain to debug.
Imho, it's down to one's personal preferences whether this should be kept or not.

Reply:
Mainly to follow the pattern set by other entries here, i.e. as a way to document the default more than anything. I agree the value will likely never be used. 

I don't think the values in `.env` are ever used, since `edx/configuraiton` supplies values in the environment, and for development and testing different files are used. I am fully in favour of using the `.env` as documentation and removing the fallback values here. We can instead just mention that the default values will be read from the `.env` file. 


These values are likely to be moot anyway with the change to JavaScript-based environment configuration: https://github.com/edx/frontend-build/pull/200

Reply:
Thanks for the clarification - the other fallback values are mostly `null` so that's why I was wondering.